dump_shc.js

```diff
@@ -27,15 +28,23 @@ console.log("-----");
 console.log(scannedJWS);
 console.log("-----");
 
+const header = getJWSHeader(scannedJWS);
 console.log("JWS Header");
-console.log(base64url.decode(scannedJWS.split(".")[0]));
+console.log(header);
 console.log("-----");
 
-verifyJWS(scannedJWS).then(
+verifyJWS(scannedJWS, header.kid).then(
   function (result) {
-    return decodeJWS(scannedJWS).then((decoded) => console.log(decoded));
+    return decodeJWSPayload(result.payload).then(
+      (decoded) => {
+        console.log(JSON.stringify(decoded.vc.credentialSubject.fhirBundle.entry, null, 2));
```

Comment:
```suggestion
        console.dir(decoded.vc.credentialSubject.fhirBundle.entry, {depth:null, compact:false});
```
This fixes the same problem but retains the coloured output.

Reply:
Thanks, that's indeed better -- I'll take care of that in my next round of revisions.

Reply:
Incorporated into a3f626028d6a507219ac11834f093fe12f9521db, thanks!